nordicsemi/__main__.py

```diff
@@ -1065,6 +1065,14 @@ def ble(package, conn_ic_id, port, connect_delay, name, address, jlink_snr, flas
         name = 'DfuTarg'
         click.echo("No target selected. Default device name: {} is used.".format(name))
 
+    # Remove colons from address in case written in format XX:XX:XX:XX:XX:XX
+    if address:
+        address = address.replace(':', '')
+
+    if address and len(address) != 12:
```

Comment:
Could maybe add a regex that checks if the string only contains valid characters 0-9 and A-F.

Reply:
Regex check added